### DISCUSSION TITLE
How to discover available API

### DISCUSSION THREAD

**Discussion by dakishin** [pybricks/support#1369] [Q&A] [answered]
Hello all. I am new in python. 
Can you tell me how can I discover available classes and methods in modules on my lego spike prime device?

```
>>> help('modules')
__main__          pybricks.experimental               pybricks.pupdevices                 umath
_hub_extra        pybricks.geometry pybricks.robotics urandom
builtins          pybricks.hubs     pybricks.tools    uselect
gc                pybricks.iodevices                  uerrno            ustruct
micropython       pybricks.media    uio               usys
pybricks          pybricks.parameters                 ujson
```

**Comment by laurensvalk** [comment, marked as answer]
All modules, classes and functions are included in the documentation on the right hand side of the app. 

If you closed it, just click the book icon to reveal it again.

It even includes documentation of all the builtin micropython functions. 🙂

**Reply by dakishin** [reply]
Thanks a lot!
